mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraDeletedMessageModule.java

```diff
@@ -46,6 +46,8 @@ public CassandraDeletedMessageModule() {
                 .addPartitionKey(MAILBOX_ID, DataType.timeuuid())
                 .addClusteringColumn(UID, DataType.bigint())
                 .withOptions()
+                .comment("Denormalisation table. Allow to retrieve UID marked as DELETED in specific mailboxes to" +
```

Comment:
I would not explain the why, only the what. I would remove `to increase EXPUNGE performance` (because we won't go back here to change it once we start using it in other contexts).